src/cdn_definitions/_impl/__init__.py

```diff
@@ -52,25 +52,27 @@ def get_data(data_paths):
         _, ext = os.path.splitext(path)
 
         if path.startswith("http"):
-            return parsed(get_remote_data(path), ext)
+            return parsed(get_remote_data(path, **kwargs), ext)
 
         if os.path.exists(path):
             return parsed(get_local_data(path), ext)
     raise RuntimeError("Could not load data")
 
 
-def load_data(source=None):
+def load_data(source=None, **kwargs):
```

Comment:
May I suggest instead accepting a Session object?  e.g.

```python
def load_data(source=None, session=None):
```

With doc string mentioning something like: "if a requests.Session is provided in "session", any HTTP requests issued while loading data will use the given session."

I think it's both simpler and more powerful than the current approach.  For example it enables session reuse (avoid establishing new TCP connection every time if you need to load data from multiple sources).

Also, the approach here with **kwargs has a problem: it makes the function support arbitrary keyword arguments while sharing the same namespace as the requests.get arguments.  If it ever happens that we later on need to add some *more* optional arguments to this function, we will not be able to pick any argument names already used by requests and we will also have to somehow untangle "OK, was this argument meant for us to pass to requests.get or not?"  Some of the arguments already have names clashing with what we might want to have in future or what a reasonable reader would assume, e.g. "verify" used by requests could just as easily mean "verify that the data being loaded matches the defined schema".

Reply:
> t's a distro-specific patch, and requests installed from pypi into a virtualenv will not give you the same behavior.

Aha, now I understand why I could only reproduce the SSL error with tox. Thanks for the explanation! 

> the approach here with **kwargs has a problem: it makes the function support arbitrary keyword arguments while sharing the same namespace as the requests.get arguments.

Yeah, good point. 

Allowing the client to configure the 'verify' param was my primary motivation behind this PR. Now I realize that won't be necessary.  I don't see much use in allowing the client to configure other parameters. Perhaps allowing the client to increase the time-out or to provide headers would be useful, but I don't have evidence to support that either would be necessary. I'll omit additional parameters for now, unless you suggest otherwise.